BRANCHES.md

```diff
@@ -6,12 +6,13 @@ At any point in time, we have a number of maintained branches, currently consist
   this always contains the latest release, including all publicly available
```

Comment:
DO we want to change `master` to `main` at some point?

Reply:
Maybe. I wondered about `latest`. Perhaps we could introduce it, document it here instead of `master`, but quietly keep `master` around for a while?

Reply:
`main` seems to be quite common these days. Keeping `master` around until `4.0` seems reasonable

Reply:
`main` is common, but typically used for what we would call `development`, I think.

Reply:
From https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/:

> Git maintainers are actively working toward a permanent change for the default name from `master` to `main`.

Reply:
Huh? `master` and by extension `main` are IME used to hold releases. Development work happens on another branch, exactly what we call `development`. See for example https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

Reply:
> master and by extension main are IME used to hold releases.

Uh, are they? Offhand I can't name a project other than Mbed TLS that doesn't treat master (or main for more recent projects) as the branch where development happens.

> https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

People use git-flow?